homeassistant/components/nextbus/sensor.py

```diff
@@ -34,6 +35,26 @@
 )
 
 
+async def async_setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
+    """Initialize nextbus import from config."""
+    _LOGGER.warning(
```

Comment:
The warning should be replaced with an issue in the issue registry.
Look for `deprecated_yaml` in the code base.